python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -17,6 +17,107 @@
 shotgun_model = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_model")
 shotgun_view = sgtk.platform.import_framework("tk-framework-qtwidgets", "shotgun_view")
 
+from .ui.widget_publish_history import Ui_PublishHistoryWidget
+
+class PublishHistoryWidget(QtGui.QWidget):
+    """
+    Simple list item widget which hosts a square thumbnail, header text
+    and body text. It has a fixed size. Used with SgPublishHistoryDelegate
+    and used in the right hand side history UI. 
+    """
+    
+    def __init__(self, parent):
+        """
+        Constructor
+        
+        :param parent: QT parent object
+        """
+        QtGui.QWidget.__init__(self, parent)
+
+        # make sure this widget isn't shown
+        self.setVisible(False)
```

Comment:
This code is really old (i just moved it from the qtwidgets code as part of this pull request!) so I don't even fully remember why it's done this way. :) Do we have a list of shotgunutils/qtwidgets v3 stuff we should be doing? If not, let's huddle and get that list going - great opportunity here to fix some stuff in v2 which weren't ideal!